### DISCUSSION TITLE
your the goat !

### DISCUSSION THREAD

**Discussion by GoldieLeGenie** [daye10/dokkan-battle-farm-bot-app#4] [General]
thanks a lot for this !

**Comment by daye10** [comment]
de rien, meme s'il reste bcp d'ameliorations a faire !

**Reply by GoldieLeGenie** [reply]
Franchement tu gères ! j’avais bossé sur un bot aussi mais j’ai laissé de côté à cause de l’encryption des stages

**Reply by daye10** [reply]
pareil j'ai eu pas mal de galeres avec... certaines cles d'encryption ont ete mis a jour depuis la version open source

**Reply by GoldieLeGenie** [reply]
Yep exactement, d’ailleurs si tu as un discord possible de d’ajouter ?

**Reply by daye10** [reply]
Oui bien sûr
